web-console/src/views/task-view/tasks-view.tsx

```diff
@@ -464,12 +476,13 @@ ORDER BY "rank" DESC, "created_time" DESC`);
           {
             Header: 'Status',
             id: 'status',
-            accessor: (row) => row.spec.suspended ? 'Suspended' : 'Running',
-            Cell: row => {
-              const value = row.value;
+            width: 300,
+            accessor: (row) => {
+              console.log(row);
```

Comment:
`console.log`

Also

I think accessor should not return a react element but a string
Save the rendering for the `Cell`